app/views/landing_page/landing_page.erb

```diff
@@ -35,6 +35,10 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
+  <% if render_analytics %>
```

Comment:
This is quickly approaching nitpicking but I assume the only purpose for the `render_analytics` local is to make preview work at editor side where the partial is not available? This could be accomplished with slighty less noise by having an opposite boolean, `skip_analytics` that doesn't need to be passed in at all for default behavior (== render analytics). You can test for local existing with: `local_assigns.has_key?(:skip_analytics)`